apps/dashboard/app/controllers/support_ticket_controller.rb

```diff
@@ -17,6 +17,10 @@ def new
     @support_ticket = support_service.default_support_ticket(params)
 
     render get_ui_template
+
+  rescue StandardError => e
+    logger.error "Could not render support ticket page. Error=#{e}"
+    redirect_to root_url, :flash => { :alert => t('dashboard.support_ticket.generic_error', error: e) }
```

Comment:
Can this be a more specific error? The idea I had in #2992 was to say specifically it's misconfigured. I think that's what we need (along with the redirect) is a specific message saying it's not configured correctly.

Reply:
will update

Reply:
Fixed. Let me know if this is what you had in mind

Reply:
Ummm, not quite. I had 

`"support_ticket is misconfigured. 'email' or 'rt_api' sections are required."`
You may even go further with 
`"support_ticket is misconfigured. 'email' or 'rt_api' sections are required in the configuration YAML."`

`service class` is a bit esoteric. I could be persuaded into even including a documentation link. The gist is basically - we have to be super explicit and leave nothing to interpretation. And **even then** we'll get tickets for the same, but maybe not as many (🤞).

Reply:
(jaja) updated again. Added a more specific error message that will be shown to the user

Reply:
That'll work just fine - thank you so much!